CryptoWorld.News/CryptoWorld.News.Server/Program.cs

```diff
@@ -14,6 +21,43 @@
 builder.Services.AddDbContext<ApplicationDbContext>(options =>
     options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
 
+// Add JWT authentication
+var jwtSettingsSection = builder.Configuration.GetSection("JwtSettings");
+builder.Services.Configure<JwtSettings>(jwtSettingsSection);
+
+var jwtSettings = jwtSettingsSection.Get<JwtSettings>();
+var key = Encoding.ASCII.GetBytes(jwtSettings.Secret);
```

Comment:
What is the reason of using Encoding.**ASCII**.GetBytes() and Encoding.**UTF8**.GetBytes in the account service?